config/redirects.yml

```diff
@@ -287,7 +287,11 @@ redirects:
   "/life-as-a-teacher/real-life-experiences/(*story)": "/my-story-into-teaching"
   "/funding-and-salary(*section)": "/funding-your-training"
   "/regional-make-a-difference-in-(*area)": "/"
-
+  "/how-to-apply(*method)": "/steps-to-become-a-teacher"
```

Comment:
/tips-on-applying-for-teacher-training might be a better one here

Reply:
Same as well for this one? `/premier-plus-adviser(*tip)`

Reply:
Think that one might be best suited as /three-things-to-help-you-get-into-teaching. I just used wayback machine to check its origins, it's solely focussed on TTAs

Reply:
Sounds like someone who [advises on hotel deals at Premier Inn](https://www.premierinn.com/gb/en/sleep/premier-plus.html) 😅